pkg/controller/atlasconnection/atlasconnection_controller.go

```diff
@@ -345,6 +347,40 @@ func (r *MongoDBAtlasConnectionReconciler) deleteDBUserFromAtlas(conn *dbaas.Mon
 	return nil
 }
 
+// getOwnedConfigMap returns a configmap object with ownership set
+func getOwnedConfigMap(connection *dbaas.MongoDBAtlasConnection, connectionStringStandardSrv, connectionStringStandard string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "atlas-connection-cm-",
+			Namespace:    connection.Namespace,
+			Labels: map[string]string{
+				"managed-by":      "atlas-operator",
+				"owner":           connection.Name,
+				"owner.kind":      connection.Kind,
+				"owner.namespace": connection.Namespace,
+			},
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					UID:                connection.GetUID(),
+					APIVersion:         "dbaas.redhat.com/v1alpha1",
+					BlockOwnerDeletion: ptr.BoolPtr(false),
+					Controller:         ptr.BoolPtr(true),
+					Kind:               "MongoDBAtlasConnection",
+					Name:               connection.Name,
+				},
+			},
+		},
+		Data: map[string]string{
+			dbaas.ConnectionStringsStandardSrvKey: connectionStringStandardSrv,
+			dbaas.ConnectionStringsStandardKey:    connectionStringStandard,
+		},
```

Comment:
I think you need to add provider, type as well in configmap. cc @jeremyary

Reply:
@redhatHameed @jeremyary The ADR says "It also specifies ServiceBindingType which is used by the Service Binding Operator. ". So should we add "ServiceBindingType", or "type" to the configmap? Also do we need to add provider to the configmap as well?

Reply:
What I understood the all the binding should contain the below value for each service-type . 
<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-b1496daa-7fff-2f87-57a2-5eb84343ab7f"><div dir="ltr" style="margin-left:0pt;" align="left">

Key Name | Purpose | Value
-- | -- | --
type | Meta field - used to match service | mongo
username | Username used to login | (string)
password | Password user to login | (string)
host | Database host | localhost:433 (string)
port | Database port | 433 (string)
provider | - human-readable value for information only. The value represents the system that is the provider of the configuration (type of operator etc.). Provider field will be not included in definitions | (string) 


</div><br /></b>

username and password pare are part of secret others params are part of configmap, and type will be changed accordingly for every service. @jeremyary correct me if I am wrong. Thanks

Reply:
I updated the code and included the type and provider to the configmap as discussed. In addition, for mongodb, we only provide the connection strings (one for standard, and one for standardSrv).

Reply:
@jianrongzhang89 how we expose MongoDB to apps is TBD based on research Sandip is doing, and potential coordination with the SBO and Quarkus teams.

Reply:
@bmozaffa yes in this morning's meeting we decided to use the connectionstrings for now. Once the final decision is done we can submit another pr if changes are needed.